src/Sql/Sql/Sql.csproj

```diff
@@ -21,12 +21,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.48.0-preview" />
+    <Reference Include="D:\src\azure\azure-sdk-for-net\artifacts\bin\Microsoft.Azure.Management.Sql\Debug\netstandard2.0\Microsoft.Azure.Management.Sql.dll" /> 
```

Comment:
Please revert the sdk change. If you want to deliver a upcoming feature, please wait for the latest SDK available in public or use local feed (not recommended).